### PR TITLE
Addition of validate.py for header validation

### PR DESCRIPTION
**Pull request by simranmadhok**
- Creation of validate.py file (within Venter app)
   - to perform header validation of the file uploaded by the user and the headers list of the logged-in user's organisation
   - to eliminate long LOC required for setting up two sets to perform header validation
   - to counter error: UnicodeDecodeError: 'utf8' codec can't decode byte 0xa5 in position 0: invalid start byte
   - to perform validation of form fields in the future instead of cluttering forms.py file (eg: Phone number field validation etc) 

- In views.py, the csv_file_header_validation() function is being invoked from validate.py where the file instance and request are passed are two arguments to the called function.

- Addition of /MEDIA file in .gitignore

- Removed extra form error message display (upload_file.html) to render form field (csv_file) manually in order to prevent duplication of form field errors

### CODE REVIEW

File: Venter/validate.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+from Venter.models import Header
+
+def csv_file_header_validation(uploaded_csv_file, request):
+    # extracting and converting the header from bytes to string format
```
Comment (Chintan2108) [inline Venter/validate.py]: @simranmadhok Please use a docstring for the function specification.

Comment (simranmadhok) [reply Venter/validate.py]: Yes, I am currently working on the documentation for the same on a separate 'documentation' branch

Comment (Chintan2108) [reply Venter/validate.py]: @simranmadhok Unfollowing the documentation branch practice from now on, yes?

Comment (simranmadhok) [reply Venter/validate.py]: Yes

**Pull request closed without merging**